map/terrain.txt

```diff
@@ -1,10 +1,10 @@
 terrain = 64
 
 
-categories =  {
+categories = {
 
 	############################
-	##### VANILLA TERRAINS ##### {
+	##### VANILLA TERRAINS #####
```

Comment:
All of these brackets in the comments are intentional -- if you setup your text editor correctly, it allows you to collapse items. They need to be kept in

Reply:
added the brackets again and also one #} at the end to have an even number